app/client/rest/channels.ts

```diff
@@ -272,7 +272,7 @@ const ClientChannels = (superclass: any) => class extends superclass {
     };
 
     viewMyChannel = async (channelId: string, prevChannelId?: string) => {
-        const data = {channel_id: channelId, prev_channel_id: prevChannelId};
+        const data = {channel_id: channelId, prev_channel_id: prevChannelId, collapsed_threads_supported: true};
```

Comment:
how do you know `collapsed_threads_supported` is supported?

Reply:
consider renaming the flag to indicate that the client supports CRT and is nothing to do with the server, does it require server changes, if yes, reconsider changing it there, naming should give a clear indication of what it is, currently got confused with server support and not client.